app/models/manageiq/providers/cloud_manager/server_group.rb

```diff
@@ -0,0 +1,66 @@
+class ManageIQ::Providers::CloudManager::ServerGroup < ::ServerGroup
+  has_many :vms, dependent => :nullify
+
+  def self.class_by_ems(ext_management_system)
+    ext_management_system&.class_by_ems(:ServerGroup)
+  end
+
+  def self.create_server_group_queue(ems_id, options = {})
```

Comment:
The usual method prototype for this is:
```suggestion
  def self.create_server_group_queue(userid, ext_management_system, options = {})
```